### PR TITLE
CODING.md,CONTRIBUTING.md: describe expected format of commit messages

### PR DESCRIPTION
**Pull request by pedronis**
as we use rebase now to merge ideally all the commit from branches should follow the expected format

### CODE REVIEW

File: CODING.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ We write them using [spread](https://github.com/snapcore/spread). Generally all
 
 * Large moving of code around and changes to code placement might also be better done separately.
 
-* PR summaries are expected to be of this form:
+* PR summaries and the first line of commit messages are expected to be of this  form:
```
Comment (ZeyadYasser) [inline CODING.md:195]: minor nit pick: there is an extra space after `this`

**Review by ZeyadYasser — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -144,8 +144,9 @@ Two approvals are required for a PR to be merged. A PR can then be merged into t
 
 On merging, you can rework the branch history as you see fit. Consider
 squashing commits from the original PR with those made during the review
-process, for example. A [force push][git-force] will be required if you do
-this.
+process, for example. Also ideally commit messages should follow the format
```
Comment (olivercalder) [inline CONTRIBUTING.md:147]: (grammar nitpick): I think the sentence should either start `Also, ideally, commit ....` or `Ideally, commit ....`

**Review by olivercalder — CHANGES_REQUESTED** [review]
Tiny grammar nitpick, otherwise looks great. Thanks!

**Review by olivercalder — APPROVED** [review]
Looks great, thanks for making the changes!

**Review by ZeyadYasser — APPROVED** [review]
Thank you!

File: CONTRIBUTING.md (hunk 2 above)
Comment (degville) [reply CONTRIBUTING.md]: Yeah, I think you're right. But even more nitpicky - having both _ideally_ and _should_ in the same sentence isn't idea. I'd probably just make this `Commit messages should follow...`

**Review by degville — APPROVED** [review]
Looks great, thank you!

**Comment by degville** [comment]
Created #13058 in relation to this.

**Pull request merged**